bin/cheribsdtest/cheribsdtest_util.c

```diff
@@ -119,3 +120,9 @@ cheribsdtest_success(void)
 	ccsp->ccs_testresult = TESTRESULT_SUCCESS;
 	exit(0);
 }
+
+void
+cheribsdtest_set_expected_si_addr(void *addr)
+{
+	ccsp->ccs_si_addr_expected = addr;
```

Comment:
style(9) wants an extra carriage return after the opening ```{``` and before any code starts, even if there are no local variable declarations.

Reply:
Ah, OK!

Reply:
Done and squashed.

Reply:
It was relaxed back in mid-2019 to be optional